.github/workflows/pull-request.yaml

```diff
@@ -276,15 +276,10 @@ jobs:
           cd cli
           go build -o ./tracetest main.go
 
-      - name: Build Current CLI
-        run: |
-          cd cli
-          go build -o ./tracetest main.go
-
       - name: Install CLI intergation version (latest main)
         run: |
           export GOPATH=$(go env GOPATH)
-          go install -v github.com/kubeshop/tracetest/cli@latest
+          go install -v github.com/kubeshop/tracetest/cli@main
```

Comment:
We were installing the latest released version instead of the latest main version.